### PR TITLE
Update OpenMRS application from 2.1.3 to 2.1.4

### PR DESCRIPTION
**Pull request by manutarus**
Due to [Vulnerabilities](https://www.bishopfox.com/news/2019/02/openmrs-insecure-object-deserialization/) in the current version of OpenMRS 

-Update OpenMRS application from 2.1.3 to 2.1.4
-update OpenMRS REST module from 2.22.0 to 2.24.0

### CODE REVIEW

File: roles/openmrs/templates/usr/share/tomcat7/.OpenMRS/openmrs-runtime.properties.j2 [hunk 1]
```diff
@@ -1,6 +1,6 @@
 connection.username={{ openmrs_mysql_user }}
 connection.password={{ openmrs_mysql_password }}
-connection.url=jdbc:mysql://{{ openmrs_mysql_host }}:{{ openmrs_mysql_port }}/openmrs?autoReconnect=true&sessionVariables=storage_engine=InnoDB&useUnicode=true&characterEncoding=UTF-8&zeroDateTimeBehavior=convertToNull
+connection.url=jdbc:mysql://{{ openmrs_mysql_host }}:{{ openmrs_mysql_port }}/openmrs?autoReconnect=true&sessionVariables=default_storage_engine=InnoDB&useUnicode=true&characterEncoding=UTF-8&zeroDateTimeBehavior=convertToNull
```
Comment (jasonrogena) [inline roles/openmrs/templates/usr/share/tomcat7/.OpenMRS/openmrs-runtime.properties.j2]: Could this entire string be converted into a variable?

Comment (manutarus) [reply roles/openmrs/templates/usr/share/tomcat7/.OpenMRS/openmrs-runtime.properties.j2]: Yes we could

File: roles/openmrs/templates/usr/share/tomcat7/.OpenMRS/openmrs-runtime.properties.j2 [hunk 2]
```diff
@@ -1,6 +1,7 @@
 connection.username={{ openmrs_mysql_user }}
 connection.password={{ openmrs_mysql_password }}
-connection.url=jdbc:mysql://{{ openmrs_mysql_host }}:{{ openmrs_mysql_port }}/openmrs?autoReconnect=true&sessionVariables=default_storage_engine=InnoDB&useUnicode=true&characterEncoding=UTF-8&zeroDateTimeBehavior=convertToNull
+connection.url=
+jdbc:mysql://{{ openmrs_mysql_connection_url }}
```
Comment (jasonrogena) [inline roles/openmrs/templates/usr/share/tomcat7/.OpenMRS/openmrs-runtime.properties.j2]: @manutarus I don't think you want to have `jdbc:mysql://` before `{{ openmrs_mysql_connection_url }}`

Comment (manutarus) [reply roles/openmrs/templates/usr/share/tomcat7/.OpenMRS/openmrs-runtime.properties.j2]: Yeah, thanks for catching that

**Review by jasonrogena — APPROVED** [review]

**Pull request merged**